sources/web/datalab/static.ts

```diff
@@ -159,13 +160,50 @@ function sendUserCustomTheme(userId: string, response: http.ServerResponse): voi
 }
 
 /**
+ * Returns true if this path should return an experimental UI resource
+ * @param path the incoming request path
+ */
+export function isExperimentalResource(path: string) {
+  const experimentalUiEnabled = process.env.DATALAB_EXPERIMENTAL_UI;
+  return experimentalUiEnabled && (
+      path.indexOf('/files') === 0 ||
+      path.indexOf('/sessions') === 0 ||
+      path.indexOf('/bower_components') === 0 ||
+      path.indexOf('/components') === 0 ||
+      path.indexOf('/images') === 0 ||
+      path.indexOf('/index.css') === 0 ||
+      path.indexOf('/modules') === 0 ||
+      path === '/'
+  );
+}
+
+/**
  * Implements static file handling.
  * @param request the incoming file request.
  * @param response the outgoing file response.
  */
 function requestHandler(request: http.ServerRequest, response: http.ServerResponse): void {
   var path = url.parse(request.url).pathname;
 
+  // -------------------------------- start of experimental UI resources
+  if (isExperimentalResource(path)) {
+    console.log('Serving experimental UI resource: ' + path);
+    if (path === '/') {
+      response.statusCode = 302;
+      response.setHeader('Location', '/files');
```

Comment:
Sorry, my earlier suggestion was wrong...

This should be `path_.join(appSettings.datalabBasePath, '/files')`... since the util.datalabLink method is only in the client-side Javascript.